Kernel/Devices/DiskDevice.h

```diff
@@ -20,6 +21,7 @@ class DiskDevice : public RefCounted<DiskDevice> {
     virtual bool read_blocks(unsigned index, u16 count, u8*) = 0;
     virtual bool write_blocks(unsigned index, u16 count, const u8*) = 0;
 
+    bool is_disk_device() const { return true; }
 protected:
-    DiskDevice();
+    DiskDevice(int, int);
```

Comment:
Let's name these arguments here since it's not obvious from the type names which argument is which, and what they are.